### PR TITLE
Unify API keys and web hooks into a single admin nav header.

### PR DESCRIPTION
**Pull request by tgxworld**
@techAPJ Can you help me review this? Thanks! :smile:

### CODE REVIEW

File: app/assets/javascripts/admin/templates/api-keys.hbs [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        {{#if k.user}}
+          {{#link-to 'adminUser' k.user}}
+            {{avatar k.user imageSize="small"}}
+          {{/link-to}}
+        {{else}}
+          {{i18n 'admin.api.all_users'}}
+        {{/if}}
+      </td>
+      <td>
+        <button class='btn' {{action "regenerateKey" k}}><i class="fa fa-undo"></i>{{i18n 'admin.api.regenerate'}}</button>
```
Comment (techAPJ) [inline app/assets/javascripts/admin/templates/api-keys.hbs]: Minor nitpick: Here we should use `d-button` component instead.

```
{{d-button action="regenerateKey" actionParam=k icon="undo" label="admin.api.regenerate"}}
```

**Comment by techAPJ** [comment]
Looks good to me! 👍

File: app/assets/javascripts/admin/templates/api-keys.hbs (hunk 1 above)
Comment (tgxworld) [inline app/assets/javascripts/admin/templates/api-keys.hbs]: Good catch! :smile:

File: app/assets/javascripts/admin/adapters/web-hook-event.js.es6 [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import RESTAdapter from 'discourse/adapters/rest';
+
+export default RESTAdapter.extend({
+  basePath() {
+    return '/admin/api/';
```
Comment (ZogStriP) [inline app/assets/javascripts/admin/adapters/web-hook-event.js.es6]: Just wondering: does this work when using subfolder? (Might have to replace this with `Discourse.getUrl("/admin/api/")`)

Comment (tgxworld) [inline app/assets/javascripts/admin/adapters/web-hook-event.js.es6]: We usually don't have to worry about subfolder paths unless we're specifying the full URL. Most of the time, `Discourse.ajax` takes cares of appending the subfolder path :smile:

**Pull request merged**